build.sbt

```diff
@@ -355,6 +355,8 @@ lazy val mtags3 = project
     sharedSettings,
     mtagsSettings,
     Compile / unmanagedSourceDirectories += (ThisBuild / baseDirectory).value / "mtags" / "src" / "main" / "scala",
+    Compile / unmanagedSourceDirectories += (ThisBuild / baseDirectory).value / "mtags-shared" / "src" / "main" / "scala",
```

Comment:
I needed to change this, since we had issues compiling mtagsShared with full version and using it from mtags3. This is just a workaround to support Scala 3 better inside Metals, so this is totally fine.